server/protocols/jmap/pom.xml

```diff
@@ -279,6 +284,21 @@
                     <artifactId>bcpkix-jdk15on</artifactId>
                 </dependency>
                 <dependency>
+                    <groupId>org.javatuples</groupId>
+                    <artifactId>javatuples</artifactId>
+                    <version>1.2</version>
+                </dependency>
+                <dependency>
+                    <groupId>org.jgrapht</groupId>
+                    <artifactId>jgrapht-core</artifactId>
+                    <version>0.9.1</version>
+                </dependency>
+                <dependency>
+                    <groupId>org.jsoup</groupId>
+                    <artifactId>jsoup</artifactId>
+                    <version>1.9.2</version>
+                </dependency>
+                <dependency>
```

Comment:
Sorry, but your Maven imports are still no correct:
- you're moving 2 imports while it should be done in the other commit (+test modifier added)
- more important, you are adding here jsoup while you don't need it at all